conf/messages.cy

```diff
@@ -0,0 +1,82 @@
+common.service-name = Cyfrif Treth Busnes
+common.fullStop = .
+common.pageTitle = {0} - {1} - GOV.UK
+common.finish = Gorffen
+common.confirmAndContinue = Cadarnhau ac yn eich blaen
+common.errorSummary.heading = Mae problem wedi codi
+common.continue = Yn eich blaen
+
+base.back = Yn ôl
+
+featureSwitch.title = Feature switches
+featureSwitch.heading = Nodweddion
+featureSwitch.submit = Cyflwyno
+
+common.error.prefixTitle = Gwall: {0}
+error.boolean = Please give an answer
+error.required = Please enter a value
```

Comment:
It seems these messages are not being used in any of the views so these could be removed

Reply:
Removed